packages/static-hosting/lib/static-hosting.ts

```diff
@@ -255,20 +265,20 @@ export class StaticHosting extends Construct {
     });
     let backendOrigin = undefined;
 
-    const originRequestPolicy = new OriginRequestPolicy(
-      this,
-      "S3OriginRequestPolicy",
-      {
+    const originRequestPolicy =
+      props.defaultBehaviorRequestPolicy ||
+      new OriginRequestPolicy(this, "S3OriginRequestPolicy", {
         headerBehavior:
           OriginRequestHeaderBehavior.allowList("x-forwarded-host"),
-      }
-    );
+      });
 
-    const originCachePolicy = new CachePolicy(this, "S3OriginCachePolicy", {
-      headerBehavior: CacheHeaderBehavior.allowList("x-forwarded-host"),
-      enableAcceptEncodingBrotli: true,
-      enableAcceptEncodingGzip: true,
-    });
+    const originCachePolicy =
+      props.defaultBehaviorCachePolicy ||
```

Comment:
Should we parametrise the `allowList` or the entire object? Either way is fine, passing an array for allowed headers is what I had in mind.

Reply:
I had that initially but figured this might be better so that we can override other properties on the policies later (cookies, query strings etc).